rclpy/src/rclpy/utils.cpp

```diff
@@ -89,4 +97,109 @@ convert_to_py(void * message, py::object pyclass)
   return py::reinterpret_steal<py::object>(convert(message));
 }
 
+const char *
+get_rmw_implementation_identifier()
+{
+  return rmw_get_implementation_identifier();
+}
+
+void
+assert_liveliness(py::capsule pyentity)
+{
+  if (0 == strcmp("rclpy_publisher_t", pyentity.name())) {
+    auto publisher = static_cast<rclpy_publisher_t *>(
+      rclpy_handle_get_pointer_from_capsule(
+        pyentity.ptr(), "rclpy_publisher_t"));
+    if (nullptr == publisher) {
+      throw py::error_already_set();
+    } else if (RCL_RET_OK != rcl_publisher_assert_liveliness(&publisher->publisher)) {
```

Comment:
@gbalke nit^N: an `else` clause is redundant here, if the `if` clause condition evaluates to `true` it'll leave the function scope on `throw`.